assets/javascripts/modules/form/email-check.js

```diff
@@ -0,0 +1,45 @@
+define(['bean',
+    'utils/ajax',
+    'utils/text',
+], function (bean, ajax, textUtils) {
+
+    'use strict';
+
+    function init() {
+
+        var YOUR_DETAILS_SUBMIT_ELEM = document.querySelector('.js-checkout-your-details-submit');
+        var EMAIL_ID_INPUT_ELEM      = document.querySelector('.js-checkout-email');
+        var USER_SIGNED_IN           = document.querySelector('data-user-signedin="true"]')
+
+        // do nothing if the user is already signed in
+        if (USER_SIGNED_IN) { return; }
+
+        ajax.init({page: {ajaxUrl: ''}});
```

Comment:
Even though this is the only time you are using this at the moment I'd recommend initialising this upfront in `main.js`, similar to https://github.com/guardian/membership-frontend/blob/master/frontend/assets/javascripts/src/main.js#L56